features/org.eclipse.rcp/rootfiles/readme/readme_eclipse.html

```diff
@@ -19,7 +19,7 @@ <h1>
   <p>
     <a
       class="mozTocH1"
-      name="mozTocId299852">Release 4.26</a>
+      name="mozTocId299852">Release 4.27</a>
   </p>
   <p>
```

Comment:
Last revised Nov 15 -> Needs to be updated.

Reply:
The current file has Revised date as November, so either it has to be removed or updated based on the strategy adopted. 
The whole Read Me can be given a new look to have the least reference of versions etc.